tests/epinio_test.go

```diff
@@ -1,4 +1,4 @@
-package cli_test
+package tests
```

Comment:
This move makes sense, as we are not dealing with unit tests here since we're using the binary. Thanks :)